advanced_source/python_extension_autoload.rst

```diff
@@ -0,0 +1,170 @@
+Out-of-tree extension autoloading in Python
+===========================================
+
+What is it?
+-----------
```

Comment:
I propose to delete this since it's an extra heading after the title w/o anything in between. 
```suggestion
```